src/main/java/com/yelp/nrtsearch/server/plugins/PluginsService.java

```diff
@@ -176,9 +179,14 @@ Class<? extends Plugin> getPluginClass(List<File> jarList, String pluginClassNam
   /** Get an instance of the loaded plugin class using reflection. */
   Plugin getPluginInstance(Class<? extends Plugin> pluginClass) {
     try {
-      return pluginClass
-          .getDeclaredConstructor(new Class[] {LuceneServerConfiguration.class})
-          .newInstance(config);
+      Plugin pClass =
```

Comment:
nit: this variable is a plugin instance, not a class. Perhaps there is a more appropriate name

Reply:
Godo point.